packages/client/src/components/Form/Form.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react'
+import { Button } from '../UI/Button'
+
+type FormProps = {
+  children: React.ReactNode
+  submitText: string
+  onSubmit: () => void
+}
+
+export const Form: FC<FormProps> = ({ children, onSubmit, submitText }) => {
+  const [loading, setLoading] = useState(false)
+
+  const handleSubmit = (e: React.SyntheticEvent) => {
+    e.preventDefault()
+    setLoading(true)
+    setTimeout(() => {
+      onSubmit()
+      setLoading(false)
+    }, 2000)
```

Comment:
Не эконом на пустых стораках, логические строки должны разделены для повышения читабельности кода
```suggestion
    e.preventDefault()

    setLoading(true)

    setTimeout(() => {
      onSubmit()
      setLoading(false)
    }, 2000)
```

Reply:
понял, учту